documentation/docs/decentralized_identifiers/overview.md

```diff
@@ -9,68 +9,69 @@ keywords:
 
-An example of a DID conforming to the `iota` method specification:
+Here is an example of a DID conforming to the `iota` method specification:
 `did:iota:8dQAzVbbf6FLW9ckwyCBnKmcMGcUV9LYJoXtgQkHcNQy`
 
 ## Chapter Overview
 
-In this chapter, we will explain the basic aspects of the DID standard. We will explore the how and why of DID Documents and why IOTA is a very suitable technology to host the DID Documents and the rest of a Self-Sovereign Identity (SSI) Framework.
+In this chapter, we will explain the basic aspects of the DID standard. We will explore the how and why of DID Documents and why IOTA is a suitable technology to host the DID Documents and the rest of a Self-Sovereign Identity (SSI) Framework.
 
 ## Decentralized Identifiers
 
-A Decentralized Identifier, or DID, is a unique identifier that is tied to a subject. This subject can be anything, like a person, an organization, an IoT device, or even an object. The identifier can be used by the subject to identify themselves through a digital format, providing a basis for online identification. The identifier looks like a set of random characters that includes some prefixes to determine which standard and implementation is used:
+A Decentralized Identifier, or DID, is a unique identifier that is tied to a subject. This subject can be anything: a person, an organization, an IoT device, or even an object. The identifier can be used by the subject to identify themselves through a digital format, providing a basis for online identification. The identifier looks like a set of random characters that includes some prefixes to determine which standard and implementation is used:
 
 `did:iota:8dQAzVbbf6FLW9ckwyCBnKmcMGcUV9LYJoXtgQkHcNQy`
 
-The World Wide Web Consortium (W3C) is a well-known standardization body that has standardized how DIDs should look and work. This provides a basis for different technologies that implement the DID standard to achieve interoperability. A full list of all implementations can be found [here.](https://www.w3.org/TR/did-spec-registries/#did-methods) Please keep in mind that unfortunately most of these methods are outdated and not maintained.
+W3C is a well-known standardization body that has defined how DIDs should look and work. This provides a basis for different technologies that implement the DID standard to achieve interoperability. A full list of all implementations can be found [here.](https://www.w3.org/TR/did-spec-registries/#did-methods) Please keep in mind that, unfortunately, most of these methods are outdated and not maintained.
```

Comment:
Ah, it was spelled out in the first sentence of the intro so I took it out since it wasn't a reference that was too far away. Throwing in the link and updating the "here".